enterprise-suite/es-grafana/akka-exporter.json

```diff
@@ -1,13 +1,13 @@
 [  
   {"graphName":"Mailbox Size", "promQL":["max by (actor)(akka_actor_mailbox_size{workload=\"$workloadId\"})"]},
   {"graphName":"Mailbox Time", "promQL":["max by (actor)(akka_actor_mailbox_time_ns{workload=\"$workloadId\",quantile=\"0.99\"})"]},
-  {"graphName":"Actor Failures", "promQL":["sum by (actor) (akka_actor_actor_failure{workload=\"$workloadId\"})"]},
-  {"graphName":"Processed Messages", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
+  {"graphName":"Actor Failures per 15 Minutes", "promQL":["sum by (actor) (irate(akka_actor_actor_failure{workload=\"$workloadId\"}[15m]))"]},
```

Comment:
per 15 minutes would require `rate` instead of `irate`.  The instantaneous functions like irate just scan back from current value to previous value to determine things, the 15m window here says "go back looking for the previous value as far back as 15 minutes.  In other words, the 15 minutes is a zero order hold for 15 minutes.

Reply:
actually, a plain `rate` would still be "per second" just using all the data in a 15 minute window to compute the average per second